src/components/container/container.js

```diff
@@ -297,47 +298,53 @@ export default class Container extends UIObject {
   }
 
   playing() {
-    this.trigger(Events.CONTAINER_PLAY, this.name)
+    this.trigger(Events.CONTAINER_PLAY, this.name, this.actionsMetadata.playEvent || {})
   }
 
   paused() {
-    this.trigger(Events.CONTAINER_PAUSE, this.name)
+    this.trigger(Events.CONTAINER_PAUSE, this.name, this.actionsMetadata.pauseEvent || {})
+  }
+
+  stopped() {
+    this.trigger(Events.CONTAINER_STOP, this.actionsMetadata.stopEvent || {})
   }
 
   /**
    * plays the playback
    * @method play
+   * @param {Object} options
    */
-  play() {
-    this.playback.play()
+  play(options = {}) {
```

Comment:
Since the `container` object is now receiving additional metadata on `play`, `pause` and `stop`, wouldn't be interesting for completeness sake to allow this data to be passed through the player public API as well? Or do we just intend to allow plugin creators to be able to do so?

Ex:

```javascript
const player = new Player({ ... })

player.pause({ metadata: 'foo' })
```

Reply:
Seems like a good feature, IMO we could add this to the backlog.

Reply:
Yes, it will be nice! But maybe we can separate responsibilities in another PR, focusing on the Player Public API feature instead of this PR. What do you think, @lucasbraganca-tw ?

Reply:
Sure!